tests/plugins/mockserver/test_unix_mockserver.py

```diff
@@ -0,0 +1,92 @@
+import aiohttp
+import pytest
+
+from testsuite._internal import fixture_types
+from testsuite.daemons import service_client
+from testsuite.mockserver import classes
+from testsuite.mockserver import server
+
+
+@pytest.fixture
+def unix_mockserver(
+    _unix_mockserver: server.Server,
+    _mockserver_trace_id: str,
+):
+    with _unix_mockserver.new_session(_mockserver_trace_id) as session:
+        yield server.MockserverFixture(_unix_mockserver, session)
+
+
+@pytest.fixture(scope='session')
+async def _unix_mockserver(
+    testsuite_logger, _mockserver_reporter, pytestconfig
+):
+    async with server.create_unix_server(
+        '/tmp/mockserver.socket',
```

Comment:
use random path please, see tmpdir fixture

Reply:
Used `tmp_path_factory` here, because `tmp_path` is function-level scoped, but `_unix_mockserver` fixture scope is session